static/js/lib/site_content.ts

```diff
@@ -107,25 +107,6 @@ export const splitFieldsIntoColumns = (
 ): ConfigField[][] =>
   partition(isMainContentField, fields).filter(column => column.length > 0)
 
-/**
- * takes a ConfigField and returns an appropriate empty value
- * for that field.
- **/
-const emptyValue = (field: ConfigField): SiteFormValue => {
-  switch (field.widget) {
-  case "select":
-    if (field.multiple) {
-      return []
-    } else {
-      return null
-    }
-  case "file":
-    return null
-  default:
-    return ""
-  }
-}
-
```

Comment:
I think this was somewhat my bad, when I made some changes to `defaultForField` and made that consistent I planned to come back and remove this `emptyValue` function since it was redundant, but I forgot to do so :/